node/src/reactor/initializer.rs

```diff
@@ -233,6 +249,7 @@ impl reactor::Reactor for Reactor {
             Event::StateStoreRequest(request) => {
                 self.dispatch_event(effect_builder, rng, Event::Storage(request.into()))
             }
+            Event::ControlAnnouncement(_) => unreachable!("unhandled control announcement"),
```

Comment:
Probably worth including the actual announcement in the panic.  Same comment applies to joiner and validator reactors.

This is probably a case for a `log_or_panic`, but since we haven't agreed an approach there, I guess we can live with panicking code here.  It seems low risk... but then again, don't they always!

Reply:
The only reason I am not including this right now is because I want to be consistent with what the reactor macro does. Let me check.

Reply:
Okay, the macro also does it. I'll update the code :)

Reply:
> This is probably a case for a log_or_panic, but since we haven't agreed an approach there, I guess we can live with panicking code here. It seems low risk... but then again, don't they always!

Well, I have to point out that currently we are only handling `fatal` as a control announcement, so I am going out on a limb here in saying this in this case its probably for the best. :)